core/src/main/resources/filodb-defaults.conf

```diff
@@ -24,17 +24,17 @@ filodb {
     columns = ["metric:string", "tags:map"]
 
     # Predefined keys allow space to be saved for over the wire tags with the given keys
-    predefined-keys = ["_ns", "app", "__name__", "instance", "dc", "le", "job", "exporter"]
+    predefined-keys = ["_ws_", "_ns_", "app", "__name__", "instance", "dc", "le", "job", "exporter"]
```

Comment:
NOTE: changing predefined keys will require a clean wipe of all data.
Could you please add a warning comment above that one should only add to predefined keys.  Changing existing keys will cause incorrect tags to be reported and probably require a clean wipe, thanks.

Reply:
This is taken care of.